README.md

```diff
@@ -1 +1,75 @@
-# steampipe-mod-gitlab-insights
\ No newline at end of file
+# GitLab Insights Mod for Steampipe
+
+A GitLab dashboarding tool that can be used to view dashboards and reports across all of your GitLab resources.
+
+![image](https://hub.steampipe.io/images/mods/turbot/gitlab-insights-social-graphic.png)
+
+## Overview
+
+Dashboards/Reports can help answer questions like:
+
+- How many projects do I have access to?
+- How many issues do I have over N days old across all my projects?
+- When was a project last contributed to?
+- When was user XYZ last active?
+
+## Getting started
+
+### Installation
+
+Download and install Steampipe (https://steampipe.io/downloads). Or use Brew:
+
+```sh
+brew tap turbot/tap
+brew install steampipe
+```
+
+Install the GitLab plugin with [Steampipe](https://steampipe.io):
+
+```sh
+steampipe plugin install theapsgroup/gitlab
+```
+
+Clone:
+
+```sh
+git clone https://github.com/turbot/steampipe-mod-gitlab-insights.git
+cd steampipe-mod-gitlab-insights
+```
+
+### Usage
+
+Start your dashboard server to get started:
+
+```sh
+steampipe dashboard
+```
+
+By default, the dashboard interface will then be launched in a new browser window at https://localhost:9194. From here, you can view dashboards and reports.
+
+### Credentials
+
+This mod uses the credentials configured in the [Steampipe GitLab plugin](https://hub.steampipe.io/plugins/theapsgroup/gitlab).
+
+### Configuration
+
+No extra configuration is required.
+
+### FAQ
```

Comment:
Is this section needed?

Reply:
@misraved is this comment in reference to the `Configuration` or `FAQ` section?